### DISCUSSION TITLE
truncated albums (CD1, CD2 etc)

### DISCUSSION THREAD

**Discussion by zedebugger** [rern/rAudio#118] [Fixed Issues]
I installed rAudio 1 today, using the "rOS - Build image files with interactive process".
I was hoping an issue from the old addon-days, before +Re1, would be solved but alas it seems no.

Albums that are stored in CD1, CD2, CDxx... folder structures are truncated when inserted to playlist. Only the first cd, CD1, gets inserted in the queue. But all files from all subfolders are properly indexed and in the database because the album view shows tracks from all subfolders with their proper ID3 sequential unique numbering.

Wagner - Der Ring des Nibelungen (Thielemann) 24-96\
|
---Die Walküre 24-96
|
---CD1 #holds tracks 1-12
---CD2 #holds tracks 13-19
---CD3 #holds tracks 20-32
---CD4 #holds tracks 32-43

Only tracks from CD1, number 1-12, gets inserted into the playlist.

For example, the structure with 4 subfolders and 43 tracks total with ID3 track numbers from 1-43 but in each subfolder the files are numbered beginning with 1 as they would be on the actual cds. I presume that flips the queueing script up in some sense, and it was like this also when I was using janui's runeAudio 05-beta with your addons, but not when using janui's runeaudio 05-beta vanilla. So it's something in the "rern implementation" that creates this problem. Do you have an idea about how it can be fixed?

**Comment by klassikmann** [comment]
@zedebugger 
In the past I faced problems with multi-volume works as operas also. I decided to use a much more "natural" way to organize the layout than to use the size limitation of CDs. Sub-directories of an opera are the several acts of the opera ("Aufzüge" in Wagner operas). Before I import the opera I use a tag-editor with the additional feature to map a tag to the file name (my choice: mp3tag) to build consecutive numbered tracks starting with 1 (both tag as file name) per act (aufzug). The album tag contains the act number in addition. 

Pros: 
- Since I applied this approach I never had database or playlist problems.
- There a no more "CD breaks" in one part of a multi-volume work.

Con:
- It's an extra work of about half an hour.

Let me iluustrate that scheme with a different recording of "Die Walküre" (foobar2000 screenshots):

![1  Aufzug](https://user-images.githubusercontent.com/25383513/104837648-3451cb80-58b6-11eb-836d-2dab89411a39.jpg)
![2  Aufzug](https://user-images.githubusercontent.com/25383513/104837649-37e55280-58b6-11eb-8b16-81c711ffb5b3.jpg)
![3  Aufzug](https://user-images.githubusercontent.com/25383513/104837653-3caa0680-58b6-11eb-8922-da34568b37df.jpg)

**Comment by zedebugger** [comment]
@klassikmann 
Thanks for the suggestion! 

I think, however, that it is not feasible for me, 

because a) (if I understand correctly) that would divide each opera of e.g. three acts into three 'albums' and since my Wagner library only is around 60 operas that would clutter things too much for my liking. 

Then b) there is chamber music (e.g. The Well-tempered Clavier) where Teil I spreads over two CDs and not lend itself to this work around since you want to create artificial segmentation. 

Finally c), I am hesitant to use such a workaround when it is not a limitation of mpd, as shown by the fact that runeaudio05-beta does not suffer from this problem. 

Hoping @rern will chime in.

**Reply by klassikmann** [reply]
@zedebugger 
Maybe we can resume my suggestion at a later point in time, possibly in the context of a discussion on how classical music should best be structured digitally (layout, tags). Regarding the examples of Wagner operas or "Das wohltemperierte Klavier", my solutions don't look so bad (IMHO).

But back to your problem. In the past I noticed inconsistencies in the database handling between RuneAudio and other mpd clients (mpd is the reference). Possible causes were too much identical tag values, multi-track flac files, additional cue sheets,... But I couldn't nail down the problemes to necessary and sufficient conditions. These experiences were one trigger to establish my own directory and tagging scheme. 

To assist in solving your problem I think it's helpful to know all the tag values of the various tracks.

**Reply by zedebugger** [reply]
I think all of us classical music lovers have find our unique ways of tagging music to fit the digital age. I remain agnostic as to whether we will ever be able to unite under the same tagging umbrella. My way of tagging classical music is very simple: I use the album tag to enter the name of the work (Die Walküre (Solti); Goldberg Variations (Gould, -55)) because I tend to listen to a work and as such I find it logical and convenient to queue one (1) work. Although, for "Nibelungen" I have been forced to come up with an unorthodox solution. Foobar2000 (Windows) and Quod Libet (Debian) have allowed me to build really neat pane based navigation systems which remain my main interface to my collection, but Runeaudio+smartphone is nice for when the desktop is shut off (music is stored on NAS). Other than album I am only using track, artist, composer, track number, disc number, year, comment.

![wagfdser](https://user-images.githubusercontent.com/65556508/104963047-ff3e9980-59d9-11eb-9fa8-42be8944c02a.jpg)

Perhaps I misunderstand your tagging suggestion, but I really would not like the prospect of segmenting all my operas into one act per album tag. Just look at the mess I would get for Wagner if there was per-act tagging:

![wagner](https://user-images.githubusercontent.com/65556508/104961924-a66e0180-59d7-11eb-9f3f-ec75ac321b05.jpg)

I presume one way out of this dilemma could be the rewrite the filenames of the entire library so that filename numbering follow ID3 track numbers and do not reset for each folder. I used to do that, but then a few years back I started using the disc number tag instead and realized I did not have to rename filenames any more. I kind of liked that. I'd still be interested in to see if @rern could not find a solution to this, rather than having to rely on a work around.

also @klassikmann, how would you deal one act ending in the middle of a cd and then the next act spanning over to another cd. Does your tagging system allow for that without altering the file name numbering?

**Comment by rern** [comment]
@zedebugger,

Try this to add the directory to Playlist:
```sh
mpc ls "PATH/TO/Wagner - Der Ring des Nibelungen (Thielemann) 24-96" | mpc add
```
`PATH/TO/...` - Full path of directory without `/mnt/MPD/`

**Reply by zedebugger** [reply]
'Wagner - Der Ring des Nibelungen (Thielemann) 24-96' is just one example of many. I have probably at least a thousand such multi-cd albums (by which I mean a classical work that spans more than one CD while having its tracks collected under one and the same "album" tag). Must there not be a way to set things up, so that this does not trips mpd up? I don't know if foobar2000 or gmusicbrowser or Quod Libet use mpd but none of them have this problem and I've been using them for years (and still do), before coming to Runeaudio.

**Reply by zedebugger** [reply]
@rern and @klassikmann 
Sorry for actually not trying the command. I erroneously thought it was a library database manipulation command rather than just acting on the playlist. So, here is the result.

The mpc command will populate the database correctly with all tracks from all subfolders. 

Going back to the GUI, I have narrowed down the error. Clicking and adding from the musical note icon in the album list view will work. But clicking the album title and coming to album view with the coverart and then clicking the coverart and adding from that popup menu will only load the first CD for the album in question. Hope this help you.

**Reply by rern** [reply]
@zedebugger,

Albums with different titles are not the same album. Album mode is title-path based reference while File mode is path only.

**Reply by zedebugger** [reply]
@rern 
There are no different album titles. Your command above spanned all four operas in the tetralogy, but don't let that distract you. I am referring to a situation where I have added just one of the operas to the command (e.g., NAS/music/Wagner - Der Ring des Nibelungen (Thielemann) 24-96/Götterdämmerung 24-96). Tags as per foobar2000 in the image. I tried this on several operas and the behaviour is consistent. When clicking the coverart image and adding, only tracks from the first CD gets inserted into the playlist, but when using the icon from the album list view all CDs are included.

(I'm not sure I grasp what you mean by album mode and file mode. I am "entering" from the composer home block button, I am not doing file browsing, to my knowledge)

![Clipboard01](https://user-images.githubusercontent.com/65556508/105213021-72691c80-5b4e-11eb-944a-5b0bd4d61b38.jpg)

![Clipboard01](https://user-images.githubusercontent.com/65556508/105212086-4a2cee00-5b4d-11eb-8ac6-c27111f483ec.jpg)

**Reply by rern** [reply]
Each thumbnail in Album mode has a tittle-artist / artist-title and a path to directory reference. 
See `/srv/http/data/mpd/album` and `/srv/http/data/mpd/albumbyartist` for better understanding.

PS. File mode: `SD`, `NAS` and `USB`

**Reply by zedebugger** [reply]
@rern 
Ok, I am beginning to see... I think... although I am not talking about album _mode_ as from the home block, I am talking about the album _view_ that you arrive at after having begun clicking on either artist or composer home block. But this discussion made me click on album home block in a trial database and I think I begin to see the problem.

The album mode actually has nothing to do with the ID3 tag 'album'. It is based on folder structure? Or where the coverart is extracted from? I fail to see why you would not consider this to be somewhat of a bug (it's my impression that you do not consider the issue I have raised to be a problem). 

The most logical thing must be to consider a double album to be one album consisting of two LPs/CDs, as long as the files have been tagged that way. Why introduce an artificial segmentation based on directory structure or something else? When clicking the album home block, I think most people would expect to see a list of albums _as per their tags_, would they not? 

It is also not consistent across the interface because if I click artist home block I will get the double album under one heading according to my tags, and clicking that album title brings me to a view of all the files from the two directories, but clicking the coverart on the same page only loads half of the files. If that is not a bug, at least it is not good UI. The switch to album mode when one has begun in artist or composer mode is neither transparent not called for, I think.

I struggle to see whether I am not understanding you or if it is you who are not understanding me.

???

![Untitled-1](https://user-images.githubusercontent.com/65556508/105376265-d997d700-5c09-11eb-99d7-f6c09c7857b0.jpg)

**Reply by rern** [reply]
> ... Clicking and adding from the musical note icon in the album list view will work. But clicking the album title and coming to album view with the coverart and then clicking the coverart and adding from that popup menu will only load the first CD for the album in question. ...

Post each scrrenshot of your "album view":
>---Die Walküre 24-96
|
---CD1 #holds tracks 1-12
---CD2 #holds tracks 13-19
---CD3 #holds tracks 20-32
---CD4 #holds tracks 32-43

**Reply by zedebugger** [reply]
Why do you want to see the individual discs? The problem is not there (even though I think a design flaw has crept into the design of the album mode). The problem is evident in the screenshot below, where clicking on ...Walküre entry in the list shows me all files from all 4 CDs in order but the path given is only CD1 and the files loaded to playlist from that page will only be those from CD1.

![Untitled-1](https://user-images.githubusercontent.com/65556508/105384970-1c11e180-5c13-11eb-8b6f-2e8b676ff577.jpg)

**Reply by zedebugger** [reply]
@rern 
I'm not demanding your attention to this issue, I just wonder if you did not notice that I replied with screenshots (above)?

Note how in the second screenshot, by the folder icon next to the coverart, says CD1 at the end but the files listed below are from all four CDs.

**Reply by rern** [reply]
By this:
> Why do you want to see the individual discs? The problem is not there ...

I've wasted my time enough.

**Reply by zedebugger** [reply]
> By this:
> 
> > Why do you want to see the individual discs? The problem is not there ...
> 
> I've wasted my time enough.

@rern 
As have I, apparently. Anything you could see from screenshots of the individual CDs have already been explained in text above, so I think it is perfectly reasonable to ask why you want to see the individual discs. No? It's a simple question to make sure that we are not talking past each other, since the problem I am experiencing appears in a different place in the GUI where you don't have access to individual CD-views.

Meanwhile the the problem remains in rAudio, but is non-existant in Runeaudio, foobar2000, QuodLibet, Gmusicbrowser... If you don't want to solve it, that's your choice.

![Untitled-2](https://user-images.githubusercontent.com/65556508/106264845-78ce5700-6226-11eb-9617-ce73d90a9130.jpg)

**Reply by klassikmann** [reply]
@rern @zedebugger Cool down 

@zedebugger 
For test purposes please change the album name (ID3-tag) "Die Walküre" to "Die Walküre CD1", "Die Walküre CD2" ...
Then perform a local Database update. Do the problems persist?

Why do i ask? As you know I have a different directory and tagging philosophy. For me the individual parts are called "Die Walküre (1. Aufzug)" ...
Your problem does not arise with this.

**Reply by zedebugger** [reply]
@klassikmann 
I appreciate your input. True, the problem does not persist after changing the album tags to 'xx CD1', 'xx CD2' etc, since in effect they then become different ID3-albums rather than the same ID3-album consisting of several CDs. The whole problem being that rAudio does not handle multi-CD ID3-albums well when browsing by composer or artist.

For me, however, this rename-the-album workaround will not work, since it would absolutely explode the number of albums in my collection making it more cumbersome to browse. It is also, IMO, an incorrect and non-elegant way to tag. We see why if we leave the world of classical music: Esbjörn Svensson Trio's "E.S.T. Live in Hamburg" is a 2-CD album, it's not two albums. Same goes for Bonobo's "Days to Come". 

Browse by artist, click on artist Bonobo. I see a list of all his albums according to the ID3-tag 'album'. I click on "Days to Come" which is a 2CD album. On the next page I am presented with _all the tracks_ from the two CDs. Yet from this screen there is no way to queue up the entire double album or even CD2. I can only queue up CD1. Rern's addon has been like this for years, ever since the early days of installing the add-on separately from Runeaudio's dev options-page. All this should be easy enough to reproduce. Yet it still is not clear to me whether the developer considers this a bug or not.

**Reply by klassikmann** [reply]
@zedebugger 
I started to build my digital music archive from scratch in 2016 with the very aged "official" RuneAudio 0.3 of 2014, then 0.4, 0.5 by Janui & Gearhard, rern's RuneUI enhancements, RuneAudio+R  and now I'am at rAudio-1. All these versions have one thing in common, they are MPD-based (https://www.musicpd.org). Over the years I learned a lot about the capabilities of MPD. These findings shaped my decision for the structure of my music archive as it is today.

The main structuring concept is a "musical work". This might be the contents of a single CD or multiple CDs, a part of a CD (consider a classical CD containing different "musical works"). An Example: "Bitches Brew" by Miles Davis is one "musical work", disseminated on two CDs because of contraints of the chosen data medium. The distribution of the work on several media does not matter. Therefore, the work looks like
![BitchesBrew](https://user-images.githubusercontent.com/25383513/106355632-94566200-62f9-11eb-9a19-48417c8ce1bb.jpg)

The sole exception of this rule can be found in operas where I decided to distribute the full work to the individual acts as "musical works". This decision is a practical consequence of my usual listening habit: One act in one session.

Some "works" require ad-hoc structure, mostly works by Jordi Savall (e.g. Jeanne d'Arc).

Another structural decision, I do not use most of the database capabilities of MPD:
![LibraryHomeDisplay](https://user-images.githubusercontent.com/25383513/106356263-b0103700-62fe-11eb-8bd6-a1bc3227f4f6.jpg)
Again, this is a consequence of my listening habit: I don't look for all works from a special date or by a specific artist. Most of the time I traverse the directory tree with several starting points as bookmarks (Rock, Jazz, and 5 for classical music), sometimes I use the album mode to browse the list of coverarts (as a consequence of the chosen structure I see e.g. Wagner operas with three coverart entries directing me to the corresponding "Aufzug").

Last but not least, you asked:
> _Yet it still is not clear to me whether the developer considers this a bug or not._

It's not a bug (IMHO).

**Reply by zedebugger** [reply]
@klassikmann 
I agree with you on the whole "musical work"-thing. The tags in my library are set up precisely in that manner. An ID3 album represents a work, even if it is spread out over several LPs/CDs. Classical has always been a bit of a tweak case, of course, so for simplicity's sake, I'll leave classical music aside and henceforth use examples based on 'Bitches Brew'.

My point of departure in all this is that I have been building my music library since 2004. It is vast and I have gotten used to software that does not impose its own excessively arbitrary ways that I need to conform to. It seems to me that rAudio does require a certain directory structure in order to work as expected. To me that is a no-starter and that's why I raised the issue I have in this thread in the hope that the developer would see my point. Alas.

The big image you post suggests that you are using directory browsing mode ("NAS share browsing") and I infer from it that you keep the files from the two CDs in the same folder (or maybe this is where the audacity 'stitching' kicks in?). So that's why you might not experience the problems I have. Otherwise you would get this view: 

![Screenshot_2021-02-01 rAudio1](https://user-images.githubusercontent.com/65556508/106491771-7f273200-64b7-11eb-9c67-58d3a9a72825.png)

If instead you browse by album you get this, which shows a departure from the idea that one album equals one musical work:

![Screenshot_2021-02-01 rAudio2](https://user-images.githubusercontent.com/65556508/106492026-c6152780-64b7-11eb-852d-30fb100d9205.png)

To me such a segmentation of an album is not just unnecessary, it is also not very elegant, and this segmentation appears to create problems when you browse by artist, composer or genre.

If you activate genre or artist browsing in your settings this is what you will see when you browse to Miles Davis (screenshot from artist browsing):

![Screenshot_2021-02-01 rAudio3](https://user-images.githubusercontent.com/65556508/106493024-eabdcf00-64b8-11eb-8976-7712b41d61ca.png)

And when you click on the name of the album, this is what you get:

![Screenshot_2021-02-01 rAudio4](https://user-images.githubusercontent.com/65556508/106493122-07f29d80-64b9-11eb-8256-6a00431797e4.png)

It looks good, but regrettably it's just a mirage. Without queuing up the individual tracks one by one, it's impossible to listen to the entire album from here if you have the files in subfolders for CD1 and CD2. This page only permits queuing up CD1. I find that very puzzling and non-intuitive since the page does _list_ the tracks from both CDs . I hardly think it is intended functionality. Maybe this is a limitation of MPD because I have not seen it in other music players. Whether it is a limitation that rern simply has chosen to accept or a bug in the rAudio implementation I guess we will not know since s/he left the thread without addressing this issue.

As a point of curiosity, I am also puzzled regarding how you square the concept of an album being a musical work with saying that you don't think it's a bug, when I describe a situation where you can't queue up more than half of the album from a page that lists the entire album? It's like having a turntable that will only play side A of an LP.

**Reply by klassikmann** [reply]
Two topics

1. Explanation what I do to prepare a multi-volume musical work as "Bitches Brew":
After ripping both CD1 I have two files for CD1 with track numbers 1 and 2 and filenames starting with 01 and 02. Ripping of CD2 generates four files with track numbers 1 to 4 and filenames starting with 01, 02, 03, 04. Then I load the four files of CD2 into a tag editor (I use mp3tag) to renumber the track numbers and filenames starting with 3. Last step is to load all six files (filenames from 01 to 06, track numbers from 1 to 6) into a tag editor (for this step I use foobar2000) to set the total tracks value to 6 and to delete unnecessary tags or edit some of them.
This "staging" step lasts about 10 minutes for an album like "Bitches Brew". But for my last investment, the complete organ works by Johann Sebastian Bach in 24/96 separated on 16 "CDs" I need two full evenings until I had the structure as you can see in the attachment.
[BachOrgel.txt](https://github.com/rern/rAudio-1/files/5910997/BachOrgel.txt)

2. To decide the question bug or constraint I recommend to use an alternative mpd-client, perhaps most suitable is mpc already installed (see https://www.musicpd.org/doc/mpc/html/) or another client mentioned in https://www.musicpd.org/clients/. If you detect the same behaviour your problem looks like a mpd constraint (my guess) otherwise it "smells" like a bug.

**Reply by zedebugger** [reply]
@klassikmann 
1. In mp3tag your have the option to add total number of tracks when you generate track numbers. You also have View -> Extended Tags that might (?) allow you to do in mp3tag everything you do in foobar.

2. It's most certainly a bug since you can queue up the two subfolders from the preceding page's options, but not from the page that lists all tracks from all subfolders. Alternatively its misleading GUI design. I'll just use the option on the preceding page in the future. My objective was to point out the bug/misleading GUI issue, if rern doesn't care, that's their choice.

**Reply by klassikmann** [reply]
@zedebugger 
ad 1. I know I use a very small subset of mp3tag only. But I have a well established workflow with foobar2000 and mp3tag in preparing new content. 

ad 2. >_It's most certainly a bug ..._
I don't subscribe to this view. Your issue remains in the state of limbo as long as you do not try to reproduce the issue using an alternative mpd-client. If you detect a different behaviour, the behaviour you expect as the correct one, than "it's a bug". If not, rern is not the contact person for you. Then you have to address the issue to the maintainer of mpd in the forum https://forum.musicpd.org/ or you write a bug report in https://github.com/MusicPlayerDaemon/MPD/issues

**Reply by zedebugger** [reply]
@klassikmann 
> ad 2. >_It's most certainly a bug ..._
> I don't subscribe to this view. Your issue remains in the state of limbo as long as you do not try to reproduce the issue using an alternative mpd-client. 

Fine with me, we don't have to agree. But, and even though this sounds a bit pretentious of me, I am actually flummoxed that _anyone at all_ would not agree with me on this issue. Okay, it may not necessarily be a bug _per se_, but if it is not, it is certainly misleading GUI-design and things would only get better if it was addressed. 

There is no need to compare with other mpd-clients. If someone designs a car with two steering wheels but only one works, I don't need to look at how other cars have been designed to say that the first car is badly designed because it makes it look like there is functionality that is not there.

But be that as it may, I think we are reaching the end of this discussion and I thank you for engaging civilly with the issue.

Enjoy your David Goode purchase! I will take a 14 min break from the computer and listen to this:

![bach](https://user-images.githubusercontent.com/65556508/106902273-ebdd3f00-66f8-11eb-85e3-32eabaacdf4a.jpg)

**Comment by klassikmann** [comment]
@zedebugger 
First of all: Did you try out the mpc command provided by rern? If mpc delivers what you want then we have evidence of a issue in rAudio.

Now back (for a last time in this context?) to the discussion about how to structure classical music.
Your Wagner collection is even more then impressive! I'm quite convinced that you are not interested in to tackle again with all of these recordings to establish another layout only. 

To demonstrate my approach an excerpt of my structure:
...
    ├── Der Fliegende Holländer
    │   └── Pinchas Steinberg + ORF Symphonieorchester
    │       ├── 1. Aufzug
    │       ├── 2. Aufzug
    │       └── 3. Aufzug
    ├── Der Ring des Nibelungen
    │   ├── Boulez, Pierre
    │   │   ├── 1 Das Rheingold
    │   │   ├── 2 Die Walküre
    │   │   │   ├── 1. Aufzug
    │   │   │   ├── 2. Aufzug
    │   │   │   └── 3. Aufzug
    │   │   ├── 3 Siegfried
    │   │   │   ├── 1. Aufzug
    │   │   │   ├── 2. Aufzug
    │   │   │   └── 3. Aufzug
    │   │   └── 4 Götterdämmerung
    │   │       ├── 1. Aufzug
    │   │       ├── 2. Aufzug
    │   │       └── 3. Aufzug
    │   ├── Janowski, Marek (1980-83)
    │   │   ├── 1 Das Rheingold
    │   │   ├── 2 Die Walküre
    │   │   │   ├── Aufzug 1
    │   │   │   ├── Aufzug 2
    │   │   │   └── Aufzug 3
    │   │   ├── 3 Siegfried
    │   │   │   ├── Aufzug 1
    │   │   │   ├── Aufzug 2
    │   │   │   └── Aufzug 3
    │   │   └── 4 Götterdämmerung
    │   │       ├── Aufzug 1
    │   │       ├── Aufzug 2
    │   │       └── Aufzug 3
    │   ├── Janowski, Marek (2012-13)
    │   │   ├── 1 Das Rheingold
    │   │   ├── 2 Die Walküre
    │   │   │   ├── Aufzug 1
    │   │   │   ├── Aufzug 2
    │   │   │   └── Aufzug 3
    │   │   ├── 3 Siegfried
    │   │   │   ├── Aufzug 1
    │   │   │   ├── Aufzug 2
    │   │   │   └── Aufzug 3
    │   │   └── 4 Götterdämmerung
    │   │       ├── Aufzug 1
    │   │       ├── Aufzug 2
    │   │       └── Aufzug 3
.....

Another aspect regarding extra work of the "one act = one directory" practice: It's not uncommon, especially in Wagner operas, that an act (Aufzug) is disseminated on two CDs. In these cases I use audacity to "stitch" together the last track of a CD with the first one of the next CD. It's cumbersome but I like it to do. But also a work nobody is willing to accomplish with an existing big music collection.

On the contrary there are CDs containing different compositions of the same or different composers. Every work is a single "album" and a single directory. Two Examples:
├── Beethoven, Ludwig van
...
│   ├── Sinfonien
│   │   ├── Sinfonie No 1 C-Dur, Op. 21
│   │   │   ├── Jan Willem de Vriend + The Netherlands Symphony Orchestra
│   │   │   └── Paavo Järvi + Deutsche Kammerphilharmonie Bremen
│   │   ├── Sinfonie No 2 D-Dur, Op. 36
│   │   │   ├── Jan Willem de Vriend + The Netherlands Symphony Orchestra
│   │   │   └── Paavo Järvi + Deutsche Kammerphilharmonie Bremen
│   │   ├── Sinfonie No 3 Es-Dur, Op. 55 'Eroica'
│   │   │   ├── Jan Willem de Vriend + The Netherlands Symphony Orchestra
│   │   │   └── Paavo Järvi + Deutsche Kammerphilharmonie Bremen
│   │   ├── Sinfonie No 4 B-Dur, Op. 60
│   │   │   ├── Jan Willem de Vriend + The Netherlands Symphony Orchestra
│   │   │   └── Paavo Järvi + Deutsche Kammerphilharmonie Bremen
.....

├── Telemann, Georg Philipp
...
│   ├── Musique de Table
│   │   └── Konrad Hünteler + Camerata des 18. Jahrhunderts
│   │       ├── Teil I
│   │       │   ├── 01 Ouverture (Suite) für zwei Flöten, Streicher und Continuo, e-Moll, TWV 55e1
│   │       │   ├── 02 Quartett für Flöte, Oboe, Violine und Continuo, G-Dur, TWV 43G2
│   │       │   ├── 03 Concerto für Flöte, Violine, Violoncello, Streicher und Continuo, A-Dur, TWV 53A2
│   │       │   ├── 04 Trio für zwei Violinen und Continuo, Es-Dur, TWV 42Es1
│   │       │   ├── 05 Solo für Flöte und Continuo, h-Moll, TWV 41h4
│   │       │   └── 06 Conclusion für zwei Flöten, Streicher und Continuo, e-Moll, TWV 505
│   │       ├── Teil II
│   │       │   ├── 01 Ouvertüre (Suite) für Oboe, Trompete, Streicher und Continuo, D-Dur, TWV 55D1
│   │       │   ├── 02 Quartett für zwei Querflöten, Blockflöte, Violoncello und Continuo, d-Moll, TWV 43d1
│   │       │   ├── 03 Concerto für drei Violinen, Streicher und Continuo, F-Dur, TWV 53F1
│   │       │   ├── 04 Trio für Flöte, Oboe und Continuo, e-Moll, TWV 42e2
│   │       │   ├── 05 Solo für Violine und Continuo, A-Dur, TWV 41A4
│   │       │   └── 06 Conclusion für Oboe, Trompete, Streicher und Continuo, D-Dur, TWV 509
│   │       └── Teil III
│   │           ├── 01 Overture (Suite) für zwei Oboen, Streicher, Continuo, B-Dur, TWV 55B1
│   │           ├── 02 Quartett für Flöte, Violine, Violoncello und Continuo, e-Moll, TWV 43e2
│   │           ├── 03 Concerto für zwei Hörner, Streicher und Continuo, Es-Dur, TWV 54Es1
│   │           ├── 04 Trio für zwei Flöten und Continuo, D-Dur, TWV 42D5
│   │           ├── 05 Solo für Oboe und Continuo, g-Moll, TWV 41g6
│   │           └── 06 Conclusion für zwei Oboen, Streicher, Continuo, B-Dur, TWV 5010

And my epilogue is the same as the prologue: What does mpc say about your Thielemann Walküre?
